mtls-examples/ruby/mtls.rb

```diff
@@ -0,0 +1,20 @@
+require 'openssl'
+require 'net/http'
+
+options = {
+  use_ssl: true,
+  verify_mode: OpenSSL::SSL::VERIFY_PEER,
+  cert: OpenSSL::X509::Certificate.new(File.read('cert.crt')),
+  key: OpenSSL::PKey::RSA.new(File.read('key.pem')),
+  ca_file: 'ca.pem'
```

Comment:
Needs a file probably, not just a filename

Reply:
It seems that for `ca_file` property of Net::HTTP class, only a path is expected....

An example here 
https://www.rubydoc.info/stdlib/net/Net%2FHTTP:ca_file

And here
https://gist.github.com/floehopper/714559 

And the docs
https://www.rubydoc.info/stdlib/net/Net%2FHTTP:ca_file

There is no Class for `ca_file` in `OpenSSL`. 
Also tried with method `File.read()` without success.

Reply:
![Screenshot 2022-07-11 at 5 29 24 pm](https://user-images.githubusercontent.com/5449175/178312729-1798604b-8e39-4032-b29e-b05cc3ca1ed3.png)

Reply:
👍🏽